pegasus/sites.v3/code.org/public/educate/professional-development-online.haml

```diff
@@ -198,6 +198,25 @@ theme: responsive_full_width
             =hoc_s(:call_to_action_start_pl)
           %a.link-button.secondary{href: CDO.studio_url("/courses/csp-2023?viewAs=Instructor"), aria:{label: hoc_s(:aria_label_call_to_action_curriculum_csp)}}
             =hoc_s(:call_to_action_explore_curriculum)
+      .action-block.action-block--one-col.flex-space-between
+        .content-wrapper
+          %p.overline=hoc_s(:module_grade_6_10_teachers)
+          %h3=hoc_s(:module_label_teaching_problem_solving)
+          %img{src: "/images/problem-solving-course-banner.png", alt: "", style: "width: 100%; height: 40%"}
+          %p
+            %strong=hoc_s(:module_label_curriculum)
+            =hoc_s(:curriculum_name_problem_solving)
+          %p
+            %strong=hoc_s(:module_label_duration)
+            =hoc_s(:module_duration_2_hrs)
+          %p
+            %strong=hoc_s(:module_label_prerequisites)
+            =hoc_s(:module_prerequisites_none)
+        .content-footer
+          %a.link-button{href: CDO.studio_url("/courses/self-paced-pl-csd-unit1-2023"), aria:{label: hoc_s(:aria_label_call_to_action_teaching_csp)}}
```

Comment:
This needs a new aria-label string that says `Start the Teaching Problem Solving and Computing professional learning program`.